src/containers/search-bar/tests/search-bar.spec.js

```diff
@@ -60,4 +60,19 @@ describe('search-bar testing', () => {
     const element = await screen.findByTestId(targetValue);
     expect(element.textContent).toBe('1');
   });
+
+test('should clear input when click on icon click', () => {
+  act(() => {
+    fireEvent.change(screen.getByPlaceholderText('searchBarPlaceholder'), {
+      target: { value: targetValue }
+    });
+  });
```

Comment:
after first fireEvent check that input value has changed

Reply:
reworked.